components/image/graphviz/history

```diff
@@ -5,3 +5,4 @@ image/graphviz/graphviz-python-27@2.40.1-2020.0.1.6
 image/graphviz/graphviz-sharp@2.40.1-2020.0.1.6
 image/graphviz/graphviz-ocaml@7.1.0-2023.0.0.2
 image/graphviz/graphviz-perl-536@7.1.0-2023.0.0.2
+image/graphviz/graphviz-perl-538@12.0.0-2024.0.0.1
```

Comment:
please add `,5.11` here.

Reply:
Why? It hasn't been done for the other obsoletions, even by you.

Reply:
Ah, you mean in this file.  Yes, right.  Mistake done more than 2.5 years ago.  We should avoid similar mistake where possible.

Reply:
We had at least one discussion whether it is needed or not. As far as I remember the oucome was that it is not needed. Even "pkg info" doesn't show it.

Reply:
Yes, it is not needed now, but it could be needed in future (although the probability is very low).  Also, it is used in almost every `history` file so for consistency it is better to add it.

... and thank you for fixing this minor issue :-).